simpleflow/swf/stats/pretty.py

```diff
@@ -269,3 +268,12 @@ def get_task(workflow_execution, task_id, details=False):
     if details:
         rows[0].append(task.get('details'))
     return header, rows
+
+
+def dump_history_to_json(history):
```

Comment:
Probably deserves a little test or docstring, I don't understand what's the expected output